src/common.h

```diff
@@ -182,4 +182,13 @@ static inline void sockaddr_set_portn(sockaddr_store_t &addr, uint16_t port)
     }
 }
 
+/**
+ * @brief
+ * @return concatenation of unix domaon socket string, useful for multiple binding.
+ */
+static inline std::string sun_path_to_string(std::string sun_path, std::string pid, std::string ifd) {
```

Comment:
1. Function content does not match the name: sun_path to string conversion is actually performed outside this function. Actual name for this function is build_client_address.
2. Use `const std::string &` to avoid unnecessary copies.
3. Pass `pid` and `ifd` as numbers and convert them inside a function.
4. Remove empty line.

Reply:
item 1 is still actual